DigitalExcellence/dex-frontend
# Bugfix/559 session timeout input lost

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce
1. Log in into Dex.
2. Create a new Project, do not click further than step 6/7.
3. Wait for 30+ minutes, or change the backend so the AUTH token is only valid for 2-5 minutes (Questions about how to do this, ask me).
4. Whenever the token is expired, a call should appear with 401 unauthorized. whenever clicked again the project should be succesfully added so the user is logged in and the session is not lost.

## Link to issue
https://github.com/DigitalExcellence/dex-frontend/issues/559

Closes: #559
